t/harness

```diff
@@ -352,13 +352,26 @@ if (@ARGV) {
 
         my $last = { par => '{' . join (',', @last) . '}/*.t' };
         @last = _extract_tests ($last);
-        push @last,
-          _tests_from_manifest($Config{extensions}, $Config{known_extensions});
 
-        push @tests, @last;
+        my ($non_ext, @ext_from_manifest)=
+            _tests_from_manifest($Config{extensions}, $Config{known_extensions}, "all");
+        push @last, @ext_from_manifest;
+
         push @seq, _compute_tests_and_ordering(\@last)->@*;
+        push @tests, @last;
 
         $rules = { seq => \@seq };
+
+        foreach my $test (@tests) {
+            delete $non_ext->{$test};
+        }
+
+        my @in_manifest_but_not_found = sort keys %$non_ext;
+        if (@in_manifest_but_not_found) {
+            die "There are test files which are in MANIFEST but are not found the t/harness\n",
```

Comment:
Missing word, should be "not found **by** the".

Reply:
Ack, I have to fix the test_pl tests, so ill push a fixup in a moment.

Reply:
Done. Thanks for the review!